src/Aquifer.API/Endpoints/Help/Response.cs

```diff
@@ -0,0 +1,9 @@
+using Aquifer.Data.Entities;
+
+namespace Aquifer.API.Endpoints.Help;
+
+public class Response
+{
+    public required List<HelpDocumentEntity> Releases { get; set; } = [];
+    public required List<HelpDocumentEntity> HowTos { get; set; } = [];
```

Comment:
We shouldn't be returning entities ever. They have circular references and can easily blow up when serialized.

Reply:
In addition to that, while not an issue in this specific case, we could potentially add something sensitive to a table, or otherwise don't want consumers to see, and by sending the whole entity down, any change to the table schema results in the response starting to include that as well. Or vice versa, say we remove a column, and everything continues to work on the back-end but the front-end blows up. If we're mapping to an object we get the compile error that we've removed something required.